frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/common/SideEffectiveActionId.java

```diff
@@ -0,0 +1,18 @@
+package com.virtuslab.gitmachete.frontend.actions.common;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.checkerframework.checker.tainting.qual.Untainted;
+
+// Let's compare action ids by reference, in case there are multiple actions with the same name going on.
```

Comment:
hmmm 🤔  on the other hand, could it not help us to to trigger the update & refresh once (and expire the previous calls)?

Reply:
Huh probably... but I'm not ready to take up such optimizations right now 😅